chihiro_todo/main.js

```diff
@@ -1,69 +1,198 @@
+        if (task.status !== 1) {
+            const doingButton = document.createElement('button');
+            doingButton.textContent = 'doing';
+            doingButton.addEventListener('click', function() {
+                changeStatus(task.id, 1);
+            })
+            newTaskElm.appendChild(doingButton);
+        }
+        if (task.status !== 2) {
+            const doneButton = document.createElement('button');
+            doneButton.textContent = 'done';
+            doneButton.addEventListener('click', function() {
+                changeStatus(task.id, 2);
+            })
+            newTaskElm.appendChild(doneButton);
+        }
+
+        const deleteButton = document.createElement('button');
+        deleteButton.textContent = 'delete';
+        deleteButton.addEventListener('click', function() {
+            deleteTask(task.id);
+        })
+        newTaskElm.appendChild(deleteButton);
+        
+        newTaskElm.appendChild(taskPersonElm);
+    }
+    const defaultPerson = function() {
+        personList.forEach((person) => {
+            defaultPersonElm = document.createElement('li');
+            defaultPersonElm.setAttribute("id", "person_" + person.id);
+            defaultPersonElm.textContent = person.name;
+            personListArea.appendChild(defaultPersonElm);
+
+            deleteButton = document.createElement('button');
+            deleteButton.textContent = 'delete';
+            deleteButton.addEventListener('click', function() {
+                deletePerson(person.id);
+            })
+            defaultPersonElm.appendChild(deleteButton);
+        });
+    }
+
+    const appendPerson = function(person) {
+        newPersonElm = document.createElement('li');
+        newPersonElm.setAttribute("id", "person_" + person.id);
+        newPersonElm.textContent = person.name
+        personListArea.appendChild(newPersonElm);
+
+        deleteButton = document.createElement('button');
+        deleteButton.textContent = 'delete';
+        deleteButton.addEventListener('click', function() {
+            deletePerson(person.id);
+        })
+        newPersonElm.appendChild(deleteButton);
+    }
+    
+    
+    const selectPersonBox = (person) => {
+        newSelectPersonElm = document.createElement('option');
+        newSelectPersonElm.setAttribute("id", "select_person_" + person.id)
+        newSelectPersonElm.value = person.id;
+        newSelectPersonElm.textContent = person.name;
+        selectedPersonId.appendChild(newSelectPersonElm);
+    }
+    
+    const defaultSelectPersonBox = () => {
+        personList.forEach((person) => {
+            const defaultSelectPersonElm = document.createElement('option');
+            defaultSelectPersonElm.setAttribute("id", "select_person_" + person.id)
+            defaultSelectPersonElm.value = person.id;
+            defaultSelectPersonElm.textContent = person.name;
+            selectedPersonId.appendChild(defaultSelectPersonElm);
+        });
```

Comment:
@chihiro-kakizaki 
この`forEach`の中で`selectPersonBox`メソッドを使ってあげると処理をまとめられそうです。
```suggestion
        personList.forEach((person) => {
            selectPersonBox(person);
        });
```